test/router.js

```diff
@@ -286,3 +286,32 @@ test(`Should return 404 on http request`, t => {
     t.end()
   })
 })
+
+test('Should pass route params to handlers', t => {
```

Comment:
Could we test also the `params` empty case? 
So we will not have regression in future

Reply:
Sounds good, I added a test with empty params.